cmd/route-controller-manager/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+	"runtime"
+
+	"github.com/openshift/library-go/pkg/serviceability"
+	"github.com/spf13/cobra"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+	"k8s.io/component-base/cli"
+
+	route_controller_manager "github.com/openshift/openshift-controller-manager/pkg/cmd/route-controller-manager"
+	"github.com/openshift/openshift-controller-manager/pkg/version"
```

Comment:
this is the last file that is not duplicated, but I can duplicate that as well.

Reply:
duplicate.